classes/Http/Action/Talk/CreateProcessAction.php

```diff
@@ -157,7 +157,7 @@ public function __invoke(HttpFoundation\Request $request): HttpFoundation\Respon
                 'flash'          => $request->getSession()->get('flash'),
             ]);
 
-            return new HttpFoundation\Response($content);
+            return new HttpFoundation\Response($content, HttpFoundation\Response::HTTP_BAD_REQUEST);
```

Comment:
I would not throw a `400` when processing forms. Browsers might misinterpret that.